### PR TITLE
[Issue 434] - R package for leonardo

### PR DESCRIPTION
**Pull request by vkumra-broad**
I tested the package manually and was able to execute a bigquery query in an R notebook.

```
install.packages("devtools")
devtools::install_github("DataBiosphere/Ronaldo", ref = "vk-issue-434", force = TRUE)
library(bigrquery)
key <-Ronaldo::getServiceAccountKey("dev")
set_service_token(key)
```

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2018, Broad Institute, Inc., Verily Life Sciences LLC
```
Comment (rtitle) [inline LICENSE:1]: This is BSD-3 right? I think that's what we're supposed to use for DataBiosphere projects.

File: R/bigQueryAccess.R [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+#' @param authEnv the environment to auth against
+#' @return the user's pet service account key
+getServiceAccountKey <-  function(authEnv = "prod") {
+  token <-
+    system("gcloud auth application-default print-access-token",
+           intern = TRUE)
+  bearer <- sprintf("Bearer %s", token)
+  project <- Sys.getenv("GOOGLE_PROJECT")
+  samHost <-
+    sprintf("sam.dsde-%s.broadinstitute.org", authEnv)  # note this is PROD; replace with dev Sam in a dev notebook
```
Comment (rtitle) [inline R/bigQueryAccess.R]: Could probably remove this comment since the env is configurable.

**Review by rtitle — APPROVED** [review]
:shipit:

File: LICENSE (hunk 1 above)
Comment (vkumra-broad) [reply LICENSE:1]: 🤷‍♂️ , I copied this directly from Leonardo

**Pull request merged**